src/components/location/LocationStyle.js

```diff
@@ -0,0 +1,93 @@
+import styled from "styled-components"
+
+export const StyledLocationPageCt = styled.main`
+  width: 100%;
+  height: 100%;
+  overflow: scroll;
+  .leaflet-container {
+    height: 30vh;
+  }
+`
+export const StyledLocationPageHeader = styled.header`
+  width: 100%;
+  min-height: 70px;
+  padding: 0 20px;
+  display: flex;
+  align-items: center;
+  font-size: 2rem;
+  border-bottom: 1px solid #d0d0d0;
+  & a {
+    display: flex;
+    align-items: center;
+    margin-right: 50px;
+    transition: all 0.15s linear;
+    cursor: pointer;
+    :hover {
+      color: orange;
+    }
+  }
+`
+export const StyledLocationPageContantTop = styled.div`
+  width: 100%;
+  min-height: 150px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  & span {
```

Comment:
it is better to have an empty new line before a child or, sub-element styles. like:
```
  justify-content: center;

  & span {
      // code
  }
```